cypress/e2e/widgets/layout.spec.ts

```diff
@@ -95,6 +95,10 @@ describe("Widget Layout", () => {
         cy.stopWebServers();
     });
 
+    it("should be set properly", () => {
+        cy.get(".mx_AppsDrawer").percySnapshotElement("Widgets drawer on the timeline (AppsDrawer)");
```

Comment:
For the record, here is the screenshot taken for the first time: https://percy.io/dfde73bd/matrix-react-sdk/builds/27613187/changed/1536053123?browser=firefox&browser_ids=36%2C38%2C39&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1024&widths=264%2C1024%2C1920

It is expected that the large grey pane lacks content, since `mx_AppTileBody` is embedded on the pane outside of `mx_AppsContainer` DOM tree. The grey pane can be manually reproduced by removing the embedded AppTileBody as below:

![0](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/1aaa580c-7f29-4b8a-bd8f-c90ba95b6723)